### ISSUE TITLE
Grey Box on screen

### ISSUE THREAD

**Issue by A16D** [MicaForEveryone/MicaForEveryone#179]
**Describe the bug**
I have actuall yfigured out what is causing this bug and i have explained in the attached picture

**Rule**
Rounded border

**Target Application**
- Application Name
- Application Version

**To Reproduce**
1. Keep Mica Open in background
2. open snipdo app in background as well
3.  Type something and snipdo will render a box to suggest its content

**Screenshots**
`<script src='https://www.yogile.com/embed/d1e2c0c8-6360-11ed-9e13-560001c42556.js?v=1.0&amp;sort=caD&amp;width=650&amp;height=438&amp;caption=false&amp;auto=false&amp;upload=false' type='text/javascript' charset='utf-8'></script>`
**Windows Version**
Windows 11 pro, 21H2

**Comment by dongle-the-gadget** [comment]
Oh. Can you open Spy++, use the window finder tool and send me its styles (including standard and extended)?

**Comment by A16D** [comment]
SO I have to download and install Visual Studio for that ? 😕

**Comment by dongle-the-gadget** [comment]
Yeah, and after all, Microsoft doesn’t provide an ARM64 version of .NET Core 3.1
